package/doc/sphinx/source/documentation_pages/transformations/boxdimensions.rst

```diff
@@ -0,0 +1 @@
+.. automodule:: MDAnalysis.transformations.boxdimensions
```

Comment:
Please also add this file to package/doc/sphinx/source/documentation_pages/trajectory_transformations.rst, down the bottom in:

```
Currently implemented transformations
-------------------------------------

.. toctree::
   
   ./transformations/translate
   ./transformations/rotate
   ./transformations/positionaveraging
   ./transformations/fit
   ./transformations/wrap

```